ct/recyclarr.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
```

Comment:
Copyright (c) 2021-2024 community-scripts ORG
Author: YOURUSERNAME

Reply:
That was a bad idea to force push. But this Done.